sentry-types/src/project_id.rs

```diff
@@ -17,26 +17,28 @@ pub enum ParseProjectIdError {
 }
 
 /// Represents a project ID.
-#[derive(Copy, Clone, Debug, PartialEq, Eq, Ord, PartialOrd, Hash, Deserialize, Serialize)]
-pub struct ProjectId(u64);
+#[derive(Clone, Debug, PartialEq, Eq, Ord, PartialOrd, Hash, Deserialize, Serialize)]
+#[serde(into = "u64", from = "u64")]
```

Comment:
IMO, we should not serialize this to `u64`, but use String there as well.
Same down below for `new` and `value`, those should take/return `&str` as well.